Extensions/glowing_follow.css

```diff
@@ -1,6 +1,6 @@
 .xglow {
 	text-shadow: 0 0 3px #FFF;
-	background: rgba(0,0,0,0.70);
+	background: rgba(0,0,0,0.70) !important;
```

Comment:
This fixes a regression introduced in #919 when I refactored the 0% and 100% styles out of the animation.